main.go

```diff
@@ -29,8 +30,13 @@ func RunServer(conf *config.Config) {
 	os.Mkdir(router.DatabaseDumpPath, 700)
 	// TODO Use config from cli
 	os.Mkdir(router.GPGPublicKeyPath, 700)
-	http.Handle("/", router.Router)
 
+	// Please make EnableSecureCSRF to false when testing locally
+	if config.EnableSecureCSRF {
+		http.Handle("/", csrf.Protect([]byte("q8satbudwexfzh2j3m5n6p8r9satcvsd"))(router.Router))
+	} else {
```

Comment:
Shouldn't this be stored in a config file or env variable instead of being hardcoded and public?